hc/settings.py

```diff
@@ -120,8 +93,8 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
-SITE_NAME = "healthchecks.io"
+SITE_ROOT = "https://hc-bs-staging.herokuapp.com/"
```

Comment:
Setting `SITE_ROOT` to the online URL makes it that checks created on the development environment are not tracked. 🤔